assets/css/plugin-check-admin.css

```diff
@@ -0,0 +1,52 @@
+/* Responsive table */
+@media only screen and (max-width: 40em) {
+	#plugin-check__submit {
+		margin-top: 1em;
+	}
+
+	table#plugin-check__categories td {
+		padding-bottom: .5em;
+	}
+
+  table.plugin-check__results-table {
+    border: 0;
+  }
+
+  table.plugin-check__results-table caption {
+    font-size: 1.3em;
+  }
+  
+  table.plugin-check__results-table thead {
+    border: none;
+    clip: rect(0 0 0 0);
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    width: 1px;
+  }
+  
+  table.plugin-check__results-table tr {
+    border-bottom: 3px solid #ddd;
+    display: block;
+  }
+  
+  table.plugin-check__results-table td {
+    border-bottom: 1px solid #ddd;
+    display: block;
+    font-size: .8em;
+    text-align: right;
+  }
+  
+  table.plugin-check__results-table td::before {
+    content: attr(data-label);
+    float: left;
```

Comment:
When using floats we'll need an RTL version too so that the layout works well when using a different text direction.

Reply:
Ok!